### PR TITLE
[containers] Fix typos

### PR DESCRIPTION
**Pull request by jlaire**

### CODE REVIEW

**Comment by tkoeppe** [comment]
Please retain only containers.tex. I'm fixing the other issues by other means.

**Comment by jlaire** [comment]
Done.

File: source/containers.tex [hunk 1]
```diff
@@ -1989,7 +1989,7 @@
 For associative containers with unique keys the stronger condition holds,
```
Comment (tkoeppe) [inline source/containers.tex]: Change to "holds:" (colon), and add to the previous paragraph: "...is positive, the following condition holds:"

**Comment by tkoeppe** [comment]
Thanks, looks good, just one small comment.

File: source/containers.tex (hunk 1 above)
Comment (jlaire) [inline source/containers.tex]: Done.

**Pull request merged**